### ISSUE TITLE
flow chart: grey/green line in background

### ISSUE THREAD

**Issue by ian-crowther** [Hederis/hederis.github.io#21]
https://www.dropbox.com/s/0ggrwxy27of6chb/Screenshot%202018-09-30%2014.09.23.png?dl=0

Eric, you'd mentioned earlier in development that things got a lot easier if the vertical "progress" bar in the background got a lot easier to deal with if it was just one color instead of a gradient. I'd prefer if the gradient stayed in just because I think it more clearly communicates a step a to step b flow, but this obviously isn't the most important part of the how the flow chart appears on the front end, so I don't think this constitutes a bug as such.

Just wondering if, at this point, there's a way to address that (either through a new asset or some CSS).

**Comment by EricWVGG** [comment]
I'll give it a try.

but there's some loopy arrows up at the top of the flowchart, I wasn't able to find those as SVG assets. Could you add them here or to the repo?

**Comment by ian-crowther** [comment]
Just added to repo. Included both isolated question mark shape as well as combined with the word/epub+pdf icons

**Comment by EricWVGG** [comment]
Timeline has the gradient now. Note that it is not possible to match the circle and rectangle effects with the gradient, unless you want to switch to a non-round rectangle (even then, the gradient won't quite line up perfectly).

**Comment by ian-crowther** [comment]
Does it help if I export svgs for those? either just the circle/round rect
or the full background progress bar with shapes included?


On October 1, 2018 at 1:43:15 PM, Eric Jacobsen (notifications@github.com)
wrote:

Timeline has the gradient now. Note that it is not possible to match the
circle and rectangle effects with the gradient, unless you want to switch
to a non-round rectangle (even then, the gradient won't quite line up
perfectly).

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Hederis/hederis.github.io/issues/21#issuecomment-425997897>,
or mute the thread

**Comment by EricWVGG** [comment]
I don't think that will work.

**Comment by ian-crowther** [comment]
Agreed that this doesn't look quite ideal. My instinct is to go back to solid green progress bar (as you originally recommended, @EricWVGG) @nelliemckesson let me know if you want to review as is.

**Comment by EricWVGG** [comment]
new solution uploaded

**Comment by ian-crowther** [comment]
@EricWVGG I think we need to go back to the solid green version of the progress bar, I'm not seeing that in the flow chart anymore. Shouldn't effect the cover screen question mark asset, but I may re-export as a solid green question mark at some point to simplify.
